smoketest/src/helper.rs

```diff
@@ -235,6 +238,41 @@ pub async fn fund_account(
 	Ok(())
 }
 
+pub async fn construct_create_agent_call(
```

Comment:
We removed these three governance calls from latest Polkadot-SDK code. We should probably remove the smoketests as well, right?

Reply:
Already removed - seems like the comment on code is outdated.